lua/lib/lib.lua

```diff
@@ -189,17 +190,47 @@ local function check_and_open_split()
 end
 
 function M.open_file(mode, filename)
+  local target_winnr = api.nvim_eval(string.format('win_id2win(%d)', M.Tree.target_winid))
```

Comment:
just a quick note that you can much more easily use these vim functions by doing
`vim.fn.win_id2win(value)` directly in lua rather than the longwinded `api.nvim_eval` calls
the same goes for `vim.fn.winbufnr`. Essentially any vim functions can be accessed using `vim.fn.func_name` (if it is a lua keyword you'd have to do `vim.fn['function'](value)`

Reply:
Yes, of course. Fixed.

Thanks for pointing it out.